packages/components/src/Notification/InlineNotification/_docs/InlineNotification.stickersheet.stories.tsx

```diff
@@ -126,9 +126,8 @@ const VARIANTS_PROPS: Array<{
       headingProps: {
         variant: "heading-6",
         tag: "span",
-        children: "Prop enforced multiline",
+        children: "Negative title",
```

Comment:
Just checking these changes to the `InlineNotification` was meant to be here? It looks like its just adding the some additional stories to document the former props to the sticker sheet so seems legit :)